python/cudf_polars/tests/expressions/test_numeric_unaryops.py

```diff
@@ -89,3 +89,9 @@ def test_log(ldf, natural):
     q = ldf.select(expr)
 
     assert_gpu_result_equal(q, check_exact=False)
+
+
+@pytest.mark.parametrize("col", ["a", "b"])
+def test_negate(ldf, col):
+    q = ldf.select(-pl.col(col))
+    assert_gpu_result_equal(q, check_exact=False)
```

Comment:
Does this work?
```suggestion
    assert_gpu_result_equal(q)
```

Reply:
Oh and maybe we should label this a feature request?

Reply:
Ah cool it does. Removed this `check_exact=False`